src/components/ProductList/ProductList.vue

```diff
@@ -0,0 +1,136 @@
+<template>
+
+  <ul class="items-container"> 
+    <ProductListItem v-for="item in items" 
+    :id="item.id" 
```

Comment:
Consider passing in the whole `item` instead of each individual field and handle what you actually need in the component itself.